### PR TITLE
wicked: Replace all empty ref files with single one

### PR DESCRIPTION
**Pull request by asmorodskyi**
wicked: Replace all empty ref files with single one

verification run : https://openqa.suse.de/tests/3050200

### CODE REVIEW

File: lib/main_common.pm [hunk 1]
```diff
@@ -1722,14 +1722,12 @@ sub load_extra_tests_filesystem {
...
-    my $wicked_dir = $basedir . 'wicked/';
-    my $suite      = get_required_var('WICKED');
-    my $type       = get_required_var('IS_WICKED_REF') ? 'ref' : 'sut';
+    my (%args)  = @_;
+    my $basedir = $bmwqemu::vars{CASEDIR} . '/tests/';
+    my $suite   = get_required_var('WICKED');
     my $exclude   = get_var('WICKED_EXCLUDE', '$a');
-    my $suite_dir = $wicked_dir . $suite . '/';
-    my $tests_dir = $suite_dir . $type . '/';
+    my $suite_dir = $basedir . 'wicked/' . $suite . '/';
```
Comment (jlausuch) [inline lib/main_common.pm]:
```suggestion
    my $suite_dir = $basedir . 'wicked/' . get_required_var('WICKED') . '/';
```

and remove line `my $suite = ...`

Comment (asmorodskyi) [reply lib/main_common.pm]: also thought about that but we use this variable also in die error message so it looks nicer to keep it

Comment (jlausuch) [reply lib/main_common.pm]: ok, right, it was hidden.

Comment (lansuse) [reply lib/main_common.pm]: why not remove $suite from die error message, and only leave $suite_dir

File: tests/wicked/ref_template.pm [hunk 2]
```diff
@@ -7,7 +7,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
```
Comment (cfconrad) [inline tests/wicked/ref_template.pm]: Copyright has 2018

File: lib/main_common.pm [hunk 3]
```diff
@@ -1760,10 +1758,22 @@ sub wicked_init_locks {
...
     wicked_init_locks();
-    my $ctx = wicked::TestContext->new();
-    for my $test (get_wicked_tests()) {
-        loadtest($test, run_args => $ctx);
-    }
+    my $ctx   = wicked::TestContext->new();
+    my @tests = get_wicked_tests();
+        for my $test (@tests) {
+            if (get_required_var('IS_WICKED_REF')) {
+                $test =~ s/\/sut\//\/ref\//;
```
Comment (cfconrad) [inline lib/main_common.pm]: Only an idea:
```suggestion
                $test =~ s'/sut/'/ref/' if (get_required_var('IS_WICKED_REF'));
```
And you can remove the outer `if (get_required_var('IS_WICKED_REF')) {}` block.

**Comment by asmorodskyi** [comment]
verification run is actually long enough to prove that provided patch do the job right , but it failing due to https://progress.opensuse.org/issues/54074 . Not sure if we should wait for full run which will pass or it is good enough to merge . WDYT @jlausuch , @cfconrad  ?

**Comment by jlausuch** [comment]
VR 
Basic: http://fromm.arch.suse.de/tests/6615
Advanced: http://fromm.arch.suse.de/tests/6613
Startstop: http://fromm.arch.suse.de/tests/6621
Aggregate: http://fromm.arch.suse.de/tests/6617

**Pull request merged**